Feature events private

santiagorodriguezbermudez opened the pull request:
## Milestone # 1

### On this milestone we accomplished:
- [x] Created a User model
- [x] Created an Event model
- [x] Created a UserEvent model to join both models with many to many associations
- [x] Created sessions for logged in users. 
- [x] Organized events according to their date (i.e. past and upcoming events)
- [x] Implemented the views for each Model accordingly

santiagorodriguezbermudez committed:
Added linters

santiagorodriguezbermudez committed:
Added user simple model

santiagorodriguezbermudez committed:
Added user session

mikethreels committed:
added event model

mikethreels committed:
linked the creator table with events

mikethreels committed:
modified show page to list all events

mikethreels committed:
added view for event index and show

mikethreels committed:
added the create and new method including view

santiagorodriguezbermudez committed:
Added UserEvent Model and the associations

mikethreels committed:
added attendees to event page

mikethreels committed:
added pluck to the attende list

mikethreels committed:
added list of events to user show

mikethreels committed:
added past and future events to user

santiagorodriguezbermudez committed:
Added event index view with past and present

santiagorodriguezbermudez committed:
Added scopes to event model

santiagorodriguezbermudez committed:
Added navigation links

santiagorodriguezbermudez committed:
Fixed rubocop offenses and stylelints errors

kobiyoyo reviewed (changes requested):
## Hi Team      ![](https://user-images.githubusercontent.com/38801423/62908122-703d4480-bd44-11e9-9115-42025ab7f341.png) 
#### Great Job so far, but there are a few more things required. 
### General
- [x] Your readme file should give detailed information on how to install and run the app.
- [x] Am still seeing the sign-up and login button while authenticated already
- [x] A link to create events should be provided.
- [x] How do I send an invitation to people? lets correct this.
- [x] The event's show page displays the list of attendants.
* [x] models have unit tests for validations and associations (Rspec) 
* [x] authentication  has integration tests (Rspec + Capybara)
* [x] event management features have integration tests (Rspec + Capybara)
### Make changes and submit another code review request.
*Happy coding* 
(Any questions Feel free to contact me on slack @kobiyoyo)

mikethreels committed:
initiated Rspec testing and adjusted the read me file

mikethreels committed:
finished adjusting readme file

mikethreels committed:
removed sign in and login buttons when logged in added logout button

mikethreels committed:
added create event to the navigation bar

mikethreels committed:
added invite option for an event

mikethreels committed:
added link to attendee

mikethreels committed:
fixed list of past and futur events

santiagorodriguezbermudez committed:
Added User unit tests

santiagorodriguezbermudez committed:
Added event validations

santiagorodriguezbermudez committed:
Added validations to UserEvents

santiagorodriguezbermudez committed:
Run validation tests

santiagorodriguezbermudez committed:
Added authentication tests

santiagorodriguezbermudez committed:
Deleted bundle in vendor path

santiagorodriguezbermudez committed:
Fixed linters on events scss

santiagorodriguezbermudez committed:
Added additional rubocop offenses

santiagorodriguezbermudez commented:
## Code Review # 1 

### On this code review we worked on the following:
- [x] Added Rspec to the project
- [x] Updated the Readme file with the program instructions
- [x] Deleted login and signup when the user is logged in
- [x] Created the link for all of the events
- [x] Created link to create an event
- [x] Added the invite option to an event
- [x] Checked the list of attendees of a single event
- [x] Implemented unit tests for validations and integration tests for models

Ispirett reviewed (approved):
Status Approved :+1: 
Great job folks! you did well with all the changes. :1st_place_medal:
